backend/api/swagger.yaml

```diff
@@ -200,34 +1112,41 @@ components:
         name:
           type: string
           example: pasta
-        catogory_id:
+        store_id:
           type: integer
           format: int64
           example: 1
         description:
           type: string
-          example: tasty pasta
+          example: "tasty pasta"
+        picture:
+          type: string
+          example: "https://i.imgur.com/1.jpg"
         price:
           type: integer
           format: int64
           example: 9999
-        storage:
+        stock:
           type: integer
           format: int64
           example: 100
-      example:
-        catogory_id: 1
-        price: 9999
-        product_id: 1
-        name: pasta
-        description: tasty pasta
-        storage: 100
+        status:
+          type: integer
+          format: int64
+          example: 1
+    ProductArray:
+      type: array
+      items:
+        $ref: '#/components/schemas/ProductInfo'
+      x-schema-name: ProductArray
     CartInfo:
       required:
       - customer_id
       - product_id
       - product_quantity
-      - store_id
+      - cart_id
+      - product_price
```

Comment:
Why cart need product_price
product_id can get price in products table

Reply:
fix